packages/core/src/util/array.ts

```diff
@@ -0,0 +1,16 @@
+const move = function(array: Array<any>, index: number, delta: number) {
+  const newIndex: number = index + delta;
+  if (newIndex < 0 || newIndex >= array.length) return; //Already at the top or bottom.
+  var indexes: number[] = [index, newIndex].sort(); //Sort the indixes
```

Comment:
did you check for the bug described here: https://gist.github.com/albertein/4496103 ?